src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -32,6 +32,24 @@ protected void SafeInit(Action init)
         {
             TransactionManager.Instance.EnsureInTransaction(DocumentManager.Instance.CurrentDBDocument);
 
+            SafeInitImpl(init);
+        }
+
+        /// <summary>
```

Comment:
Shouldn't we handle this the other way around and have a default param set to true to ensureTransaction ?